.phan/config.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * @author Patrick Jahns <github@patrickjahns.de>
```

Comment:
```suggestion
 * @author Patrick Jahns <github@patrickjahns.de>
```
:arrow_up:

Reply:
This is a copy-paste of the config from other repos, originally written by Patrick. I don't know what the rules are when doing that sort of thing. In the other repos, I have been leaving his name.